Dockerfile

```diff
@@ -0,0 +1,22 @@
+# build environment
+FROM node:16-alpine as build
+
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json ./
+COPY package-lock.json ./
+COPY yarn.lock ./
+
+RUN npm ci --silent
+RUN npm install react-scripts -g --silent
```

Comment:
no need to install this package manually It will be installed with about command